cmd/admin-config-get.go

```diff
@@ -90,8 +90,14 @@ func mainAdminConfigGet(ctx *cli.Context) error {
 	c, e := client.GetConfig()
 	fatalIf(probe.NewError(e), "Cannot get server configuration file.")
 
+	config := map[string]interface{}{}
+	e = json.Unmarshal(c, &config)
```

Comment:
What's the reason for doing this ? only checking the json ?

Reply:
Because server already puts it in JSON and to handle double marshalling..